cmd/mount.go

```diff
@@ -94,6 +94,7 @@ type mountOptions struct {
 	AttrCache      bool     `config:"use-attr-cache"`
 	LibfuseOptions []string `config:"libfuse-options"`
 	BlockCache     bool     `config:"block-cache"`
+	EntryCache     int      `config:"list-cache-timeout"`
```

Comment:
Rename this to something more meaningful, `EntryCacheTimeout`

Reply:
Corrected.